apps/mull-ui/src/app/pages/create-event/create-event.tsx

```diff
@@ -239,15 +240,7 @@ const CreateEventPage = ({ history }: CreateEventProps) => {
             errorMessage={formik.errors.description}
             svgIcon={<FontAwesomeIcon icon={faAlignLeft} />}
           />
-          <CustomTextInput
-            title="Location"
-            fieldName="location"
-            value={formik.values.location}
-            onChange={formik.handleChange}
-            hasErrors={formik.touched.location && !!formik.errors.location}
-            errorMessage={formik.errors.location}
-            svgIcon={<FontAwesomeIcon icon={faMapMarkerAlt} />}
-          />
+          <LocationInput formik={formik} />
```

Comment:
If I submit then go back to the form the location is missing in the input field.

If I do the above I can submit the form with an empty location, the validation didn't stop me.

Reply:
Could you send a gif of the situation? I'm not exactly sure what you mean

Reply:
I will show you today

Reply:
@ys-lin I can only show you after the final but it's simple to replicate when you run the apl

1. Fill in the form and submit
2. Press edit from review page
*Notice the location is empty now
3. Submit the form
*Notice the form was submitted with empty location

Reply:
Ah, I got it now, thank you!